integration-test/tests/login-form.macos.spec.js

```diff
@@ -108,49 +108,68 @@ test.describe('Auto-fill a login form on macOS', () => {
         })
     })
 
-    test.describe('when there is a single unknown username field, and one password field', () => {
-        test('the unknown field is a username', async ({page}) => {
-            await forwardConsoleMessages(page)
-            await mocks(page, {
-                unknown_username_categorization: true
+    test.describe('when there is a single ambiguous field, and one password field', () => {
```

Comment:
I think I had already given this feedback in the previous PR related to this change. Why are we using playwright tests instead of the form test suite? Playwright is way slower and it's not usually run when we iterate on the algorithm. Can't recall if there was a specific reason to do this in playwright, but if not we should probably move this test to the unit test suite.

Reply:
I remember that comment, but I don't think it was about this test case. I cannot find it here at least: https://github.com/duckduckgo/duckduckgo-autofill/pull/619. 

For this specific case, I think integration test is extremely useful as it helps validate the happy path, and adds to confidence. I can merge the asserts it with the previous test though, and in that case we wouldn't need to do additional setups for the new one.

I am looking for some real world forms right now, will update the unit tests.

Reply:
My take is that this shouldn't change anything in the happy path. What we're changing is not the interaction, but the form scoring. Form scoring is tested in the unit test suite, when we change the algo that's what we look at for regressions. Once the form is scored, everything follows as usual, there's nothing specific happening here that we need to test in playwright. That is unless we think that is not the case. But I can't think of a reason why.

By the way, you can also create the test forms you need. Just drop a new form with specific requirements in the test suite. You already have one in the playwright test, for example. But of course, if you find one in the wild, all the better. I'm sure there are a few in the bug reports. Just timebox it. If you don't find anything right away, make up a form and move on. We will find failure cases in the future and we can add forms then. Just my 2 cents.

Reply:
@GioSensation Ready to go!

Reply:
Since we're moved this to the unit test, are these tests still needed? Or can we delete them?

Reply:
Removed the integration tests around it.